legend-engine-pure-code-compiled-core/src/main/resources/core/pure/router/preeval/preeval.pure

```diff
@@ -255,6 +255,22 @@ function <<access.private>> meta::pure::router::preeval::prevalInternal(item : A
   $newItem;
 }
 
+function <<access.private>> meta::pure::router::preeval::preevalAggregateFunction(f:FunctionDefinition<Any>[1], state : meta::pure::router::preeval::State[1], extensions:Extension[*]):PrevalWrapper<Any>[1]

```

Comment:
Is there a better name for this function? It's not really aggregate function specific, but feels more about the nested lambda and open variables